deps/wrapper.cc

```diff
@@ -104,6 +121,26 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
 
     mod.method("BuildJulia", &FunctionDescriptorBuilder::BuildJulia);
     mod.method("ToString", &ToString);
+
+    // class Buffer
+    // https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/common/buffer.h
+    mod.add_type<Buffer>("Buffer")
+        .method("Data", &Buffer::Data)
+        .method("Size", &Buffer::Size)
+        .method("OwnsData", &Buffer::OwnsData)
+        .method("IsPlasmaBuffer", &Buffer::IsPlasmaBuffer);
+    mod.add_type<LocalMemoryBuffer>("LocalMemoryBuffer", jlcxx::julia_base_type<Buffer>());
+    mod.method("LocalMemoryBuffer", [] (uint8_t *data, size_t size, bool copy_data = false) {
+        return std::make_shared<LocalMemoryBuffer>(data, size, copy_data);
+    });
+
+    mod.method("put", &put);
+    mod.method("get", &get);
+
+    // mod.add_type<RayObject>("RayObject")
+    //     .constructor<const std::shared_ptr<Buffer>,
+    //                  const std::shared_ptr<Buffer>,
+    //                  const std::vector<rpc::ObjectReference>,
+    //                  bool>();
```

Comment:
We'll probably want to expose this in the future. It's not technically needed here but I started down this path so there's little harm to leaving it in in case we do end up using it